cfme/tests/infrastructure/test_host_analysis.py

```diff
@@ -104,6 +104,9 @@ def is_host_analysis_finished():
     sel.handle_alert()
 
     # Check results of the analysis
+    drift_history = test_host.get_detail('Relationships', 'Drift History')
```

Comment:
Yes, ```get_detail``` will navigate to the details page, however it cannot detect if simply a refresh is needed, so in order to speed up I used ```InfoBlock``` in next occurrences